tests/integration/networks/evm/helpers.rs

```diff
@@ -0,0 +1,94 @@
+//! Helper functions for EVM network tests
+
+use crate::integration::common::{
+    client::{CreateRelayerRequest, RelayerClient, RelayerResponse},
+    registry::TestRegistry,
+};
+use eyre::Result;
+use openzeppelin_relayer::models::relayer::RelayerNetworkType;
+use tracing::{info, warn};
+
+/// Setup a relayer for testing on a specific network
+pub async fn setup_test_relayer(
```

Comment:
Do we need to setup/create test relayer programatically?

For network tests we can simply use config file with already defined relayers to be used for testing.